<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BanqueMainWindow</class>
 <widget class="QMainWindow" name="BanqueMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BanqueMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Connexion</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Adresse IP</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditIP">
               <property name="text">
                <string>172.18.58.14</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Nom de la bdd</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditDB">
               <property name="text">
                <string>banque</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>login</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditLogin">
               <property name="text">
                <string>snir</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>mot de passe</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditMDP">
               <property name="text">
                <string>snir</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonConnexion">
           <property name="text">
            <string>Connexion à la base de données</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Affichage infos comptes</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Afficher les informations de tous les comptes</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="tableWidgetInfos"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Quitter</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>BanqueMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>559</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
